### PR TITLE
Logging support for untranslated messages

### PR DESCRIPTION
**Pull request by fprochazka**
[Closes #95]

### CODE REVIEW

**Comment by enumag** [comment]
@fprochazka In my opinion there should by just one logger property in Translator (the PSR one). Tracy should use a bridge.

**Comment by fprochazka** [comment]
@enumag I agree

**Comment by fprochazka** [comment]
Anyway, it's protected and optional, you can easily extend the Translation class, if you really must, and change the implementation.

File: src/Kdyby/Translation/Translator.php [hunk 1]
```diff
@@ -419,6 +437,20 @@ private function extractMessageDomain($message)
...
+	 * @param string $message
+	 */
+	protected function logMissingTranslation($message)
+	{
+		if ($this->psrLogger) {
+			$this->psrLogger->notice('Missing translation', [
+				'message' => $message,
+			]);
+		}
+	}
```
Comment (enumag) [inline src/Kdyby/Translation/Translator.php]: I'd probably move the panel-related code into this method as well. I mean this part.

```
if ($this->panel) {
	$this->panel->markUntranslated($message);
}
```

**Comment by janedbal** [comment]
Why is ```psr/log``` required dependency? I think it should be in suggest only.

**Comment by fprochazka** [comment]
It's just na interface, you can use whatever implementation you want.

**Pull request merged**